common/src/main/scala/react/floatingui/syntax.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package react.floatingui
+
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.ui.syntax.all.given
+
+import scala.scalajs.js
+
+object syntax:
+  // extension (tag: VdomTag)
```

Comment:
I want to add these extensions for when we want to add tooltips to dom elements that don't need to be wrapped in a `span`. However, I can't use the same names and still have default parameters because the compiler complains about `2 or more overloaded methods have default parameters`. Any suggestions?

Reply:
Low priority implicits?

Reply:
As I understand it, this isn't a problem of resolution, but a problem with how methods with default parameters are named. This results in two methods with the same name, and the other parameters don't disambiguate for some reason.

IIRC, this is one of the things the magnet pattern was designed for. I'll see how ugly that would be.

Reply:
I did not recall correctly. The magnet pattern cannot handle default parameters.